apisix/plugins/jwt-auth.lua

```diff
@@ -344,8 +341,8 @@ end
 function _M.rewrite(conf, ctx)
     local jwt_token, err = fetch_jwt_token(ctx)
     if not jwt_token then
-        if err and err:sub(1, #"no cookie") ~= "no cookie" then
-            core.log.error("failed to fetch JWT token: ", err)
+        if err then
```

Comment:
Do we still need this check?

Reply:
Seems we don't need to check `err` as func `fetch_jwt_token` doesn't return it any more.

Reply:
`fetch_jwt_token` would return `"JWT not found in cookie"`

Reply:
update